locations/open_graph_parser.py

```diff
@@ -3,17 +3,22 @@
 
 
 class OpenGraphParser:
-    @staticmethod
-    def parse(response) -> Feature:
+    def extract_properties(self, response):
         keys = response.xpath("/html/head/meta/@property").getall()
         src = {}
         for key in keys:
             if key.startswith("og:") or key.startswith("place:location:") or key.startswith("business:contact_data:"):
                 content = response.xpath('//meta[@property="{}"]/@content'.format(key)).get()
                 if content:
                     src[key.split(":")[-1]] = content
-        item = DictParser.parse(src)
+        return src
+
+    def as_item(self, properties, response):
+        item = DictParser.parse(properties)
         item["website"] = response.url
         if not item.get("ref"):
             item["ref"] = response.url
         return item
+
+    def parse(self, response) -> Feature:
```

Comment:
I think this can still be a static method so you don't have to instantiate an `OpenGraphParser` to call parse.